### PR TITLE
Feature/chat locker 54 - show room owner/creator

### PR DESCRIPTION
**Pull request by berkich**

### CODE REVIEW

File: client/views/app/room.html [hunk 1]
```diff
@@ -147,6 +147,7 @@
 											<p><i class="icon-phone"></i> {{phone}}</p>
 										{{/if}}
 										--}}
+                                                                            <span class="glyphicon-heart" title="Group Owner"></span>
```
Comment (rwakida) [inline client/views/app/room.html]: should there be a isOwner check before displaying this?  Also we used a check in the member list, but a heart here for the icon

**Comment by rwakida** [comment]
I can't view the glyphicon.  It appears like this:
glyphicon

**Comment by berkich** [comment]
If user A creates a direct message with user B at a specific classification, user B can change it to a lower level. Not sure if this was fixed in another issue

**Comment by bbrockman** [comment]
Pretty sure this was fixed before where you could not lower the classification.

**Comment by berkich** [comment]
I just tested it with the latest version in master and the results are the
same

**Pull request merged**